sdk-setup/src/mb2

```diff
@@ -309,6 +309,24 @@ COMMANDS
         accept arguments need to be sticked together with their arguments with
         '='.)
 
+    generate-key -n|--name[=<GPG key name>] -e|--email[=<GPG key email>] -p|--passphrase[=<GPG passphrase>] -i|--install
+        Run generating developer key for sign packages.
+        -n|--name[=<GPG key name>]
+        -e|--email[=<GPG key email>]
+            Name and email is part of user data needs to construct a user ID to identify key.
+         
+         -p|--passphrase[=<GPG passphrase>]
+            Passphrase for new key.
+
+         -i|--install
+            If install option enabled, key will be exported to rpm and passphrase exported to .....
+
+    key-list
+        Show GPG keys, installed in rpm
+
+    deploy-key -n|--name[=<GPG key name>]
+        Deploy all GPG key to a device with name <name>.
+
```

Comment:
The user should be able to fully control all aspects of their keys creation and so I am certain mb2 should not provide commands for key management and only allow to use existing keys - created with the tool of user's choice or supplied at organization level, etc.

mb2 may be running under the build engine as well, while the keys would be stored on host, so gpg needs to be pointed to the correct location under build engine (`--homedir "$HOME/share/.gnupg"`?).

The key should be then passed with the help of a global option instead of a subcommand. We can think about the proper naming later (it needs to play nicely with sfdk whose domain is much wider and so naming really matters there), and let's work with this meanwhile: `--signing-key <ID-or-file> --signing-passphrase[-file] <passphrase>`.

The `deploy-key` command shouldn't be added either. Similarly to key generation, key deployment is rarely done and well supported with existing tools. So it is enough to add a hint to the documentation of the `--signing-key` option, that the key may be deployed to a device with command like:

`gpg2 --export -a <ID> |mb2 run sudo rpmkeys --import -`

And equivalently with sfdk:

`gpg2 --export -a <ID> |device exec sudo rpmkeys --import -`


For that we need to fix `rpmkeys` to allow reading the key from stdin (see https://github.com/rpm-software-management/rpm/blob/bfee1410af51c1cc9724791fb8d985260a62102b/lib/rpmchecksig.c#L85).